sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -9,7 +9,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="2b8e4de42771033072324713ba8919248585d263"  # flatcar-master
+	CROS_WORKON_COMMIT="1d32bea2c0e4335d4a8f7e0ccd6a7b41da15e4a7"  # add group "render" for udev
```

Comment:
> add group "render" for udev

The comment looks strange, as it is not a branch name any more.
Are we going to then write human-readable comments that represent the latest commit in the repo, changing it every time when the repo got updated? Sounds like overkill.
I would rather just keep it as `main`.

Reply:
Makes sense - I'll update the comment when / if https://github.com/flatcar-linux/baselayout/pull/5 gets merged.

Reply:
Just to add, I guess baselayout is still `flatcar-master` and not `main`.